host/host_windows.go

```diff
@@ -80,7 +80,7 @@ func HostIDWithContext(_ context.Context) (string, error) {
 	hostID := windows.UTF16ToString(regBuf[:])
 	hostIDLen := len(hostID)
 	if hostIDLen != uuidLen {
-		return "", fmt.Errorf("HostID incorrect: %q\n", hostID) //nolint:revive //FIXME
+		return "", fmt.Errorf("hostID incorrect: %q", hostID)
```

Comment:
This one is not logic, it should be this

```suggestion
		return "", fmt.Errorf("HostID incorrect: %q", hostID)
```

What did revive report?

We made change to avoid this

- https://github.com/mgechev/revive/pull/1287

revive 1.9.0 included this

And golangci-lint v2.1.0 included it

https://golangci-lint.run/product/changelog/#v210